src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  @include flex-justify-align(center, center);
+}
+
+.stopwatch {
+  width: 500px;
+  height: 500px;
+  background-color: aqua;
+  border-radius: 50%;
+  @include flex-justify-align(center, center);
+
+  &__hands {
+    width: 400px;
+    height: 400px;
+    position: relative;
+
+    &-circle {
+      width: 20px;
+      height: 20px;
+      background-color: yellow;
+      border-radius: 50%;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-8px, -10px);
```

Comment:
Where did you get this exact pixels?
I think it should be -50%

Reply:
-10px is the actual 50% and -8px the 50% minus the half of hands-minutes (except the 0.5px - magic pixel)

Without the 2px it looks like this - https://imgur.com/a/srcmoeY